cmake/Modules/LocalRav1e.cmake

```diff
@@ -99,6 +99,10 @@ else()
 
     set(RAV1E_LIBRARIES ${Rust_CARGO_TARGET_LINK_NATIVE_LIBS})
     if(WIN32)
+        # If an item starts with "/" (e.g., "/defaultlib:msvcrt"), change the
+        # leading "/" to "-" so that the target_link_libraries() call below
+        # will treat the item as a linker flag.
+        list(TRANSFORM RAV1E_LIBRARIES REPLACE "^/" "-")
```

Comment:
The `list(TRANSFORM)` operation is new in cmake 3.12:
https://cmake.org/cmake/help/latest/command/list.html#transform

Our cmake minimum required version is 3.13, so it is safe to use `list(TRANSFORM)`.